services/wordpress/wordpress-last-update.service.js

```diff
@@ -1,16 +1,12 @@
-import dayjs from 'dayjs'
-import customParseFormat from 'dayjs/plugin/customParseFormat.js'
-import { InvalidResponse, pathParams } from '../index.js'
-import { formatDate } from '../text-formatters.js'
-import { age as ageColor } from '../color-formatters.js'
+import { pathParams } from '../index.js'
+import { parseDate, renderDateBadge } from '../date.js'
 import { description, BaseWordpress } from './wordpress-base.js'
-dayjs.extend(customParseFormat)
 
 const extensionData = {
   plugin: {
     capt: 'Plugin',
     exampleSlug: 'bbpress',
-    lastUpdateFormat: 'YYYY-MM-DD hh:mma [GMT]',
+    lastUpdateFormat: 'YYYY-MM-DD h:mma [GMT]',
```

Comment:
This was not strictly related to this task, but was something I noticed while I was working on this. The hours part of timestamps in this format omits the leading zero if it is single digit, so this should be `h` not `hh`.